app/src/main/java/com/onboarding/wheatherapp/data/entity/Main.kt

```diff
@@ -0,0 +1,7 @@
+package com.onboarding.wheatherapp.data.entity
+
+data class Main(
+     val temp: String,
+     val tempMin: String,
+     val tempMax: String
+)
```

Comment:
add empty line at the end of the file pls

Reply:
Solved!